src/saveload/group_sl.cpp

```diff
@@ -23,7 +23,7 @@ static const SaveLoad _group_desc[] = {
 	SLE_CONDNULL(2,                                                         SL_MIN_VERSION,  SLV_164), // num_vehicle
 	     SLE_VAR(Group, owner,              SLE_UINT8),
 	     SLE_VAR(Group, vehicle_type,       SLE_UINT8),
-	     SLE_VAR(Group, replace_protection, SLE_BOOL),
+	     SLE_VAR(Group, flags,              SLE_UINT8),
```

Comment:
While I think the standard somewhere says that the numeric value of `true` is 1, I'm not sure relying on old savegames to only contain 0 or 1 here is wise.

As you are doing a savegame bump anyway, maybe clear all other bits for old saves?

Reply:
Not sure if it was the case at the time, but it is tested to be non-zero and clear if so during AfterLoad. It's handled in the bulky main AfterLoad function as it also sets the flag based on company information.